packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -1,7 +1,10 @@
+
 import React from 'react';
 import PropTypes from 'prop-types';
-import ReactDatePicker from 'react-datepicker';
 import ResponsiveElement from 'terra-responsive-element';
+
+/* eslint-disable-next-line  */
```

Comment:
nit: can we specify the exact rule we are disabling?

Reply:
react-datepicker uses syntax that isn't currently supported by our linter, so it's throwing parse errors when it tries to run the linter on this file. As a result, it's always throwing lint errors, even when I specify the rules that need to be ignored.